addon/components/async-button.js

```diff
@@ -12,10 +12,17 @@ export default Ember.Component.extend({
   reset: false,
   classNames: ['async-button'],
   classNameBindings: ['textState'],
-  attributeBindings: ['disabled', 'type', '_href:href'],
+  attributeBindings: ['disableWhen', 'disabled', 'type', '_href:href', 'tabindex'],
 
   type: 'submit',
-  disabled: Ember.computed.equal('textState','pending'),
+  disabled: Ember.computed('textState', 'disableWhen', function() {
+      var textState = this.get('textState');
```

Comment:
Totally agree on the get vs this.get but the let change seems strange ... the entire component has local vars (not a single let that I can see) -confirm?